src/com/rockbite/bootcamp/util/observer/Subject.java

```diff
@@ -0,0 +1,64 @@
+package com.rockbite.bootcamp.util.observer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
```

Comment:
purpose of class unclear. Due to name probably. Why would classes subscribe to this subject. why?